devices/Bh1750fvi/Bh1750fvi.cs

```diff
@@ -10,19 +10,19 @@
 namespace Iot.Device.Bh1750fvi
 {
     /// <summary>
-    /// Ambient Light Sensor BH1750FVI
+    /// Ambient Light Sensor BH1750FVI.
     /// </summary>
     [Interface("Ambient Light Sensor BH1750FVI")]
     public class Bh1750fvi : IDisposable
     {
-        private const byte DefaultLightTransmittance = 0b_0100_0101;
+        private const byte DefaultLightTransmittance = 0b01000101;
```

Comment:
From my experience problem is the second underscore. Try with 0b_10000101

Reply:
Actually, the problem was in first one.
0b_10000101 -> invalid
0b1000_0101 -> valid